src/actions/render.js

```diff
@@ -100,25 +103,29 @@ const codeGenerationOutputPath = (basePath, platform) => {
 
 const localizationFileName = platform => {
   switch (platform) {
-    case platformKeywords.ANDROID:
+    case outputType.ANDROID:
       return "strings.xml";
-    case platformKeywords.IOS:
+    case outputType.IOS:
       return "Localizable.strings";
+    case outputType.JS:
+      return "strings.json";
   }
 };
 
 const codeGenerationFileName = platform => {
   switch (platform) {
-    case platformKeywords.ANDROID:
+    case outputType.ANDROID:
       return "Localizable.kt";
-    case platformKeywords.IOS:
+    case outputType.IOS:
       return "Localizable.swift";
+    case outputType.JS:
+      return "Localizable.ts";
```

Comment:
We'll probably need to change this to emit a JS + type-declarations file in a follow-up PR.